src/main/java/org/broadinstitute/consent/http/service/DataSetService.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.consent.http.service;
+
+import org.broadinstitute.consent.http.db.DataSetDAO;
+
+import javax.inject.Inject;
+import java.util.Date;
+
+
+public class DataSetService {
```

Comment:
Nitpick: Can you please name this `DatasetService`? The mixed casing of `DataSet` has always bugged me.

Reply:
Noted! To be honest I actually prefer Dataset as well. Would it make sense to refactor DataSet -> Dataset wherever touched moving forward?

Reply:
Yes, please!!! We love pure refactoring PRs too because they should never break any functionality.